### ISSUE TITLE
Сверточные нейронные сети

### ISSUE THREAD

**Issue by nikibor** [nikibor/Yelp-commentaries#6]
Изучить метод.
Посмотреть реализации в фреймворках.
Опробовать на данных

**Comment by nikibor** [comment]
Руководство по созданию сети в keras
- [ ] http://adventuresinmachinelearning.com/keras-lstm-tutorial/

**Comment by nikibor** [comment]
Архитектура сверточной нейронной сети
- [ ] https://www.kaggle.com/tonypoe/keras-cnn-example?scriptVersionId=589403

Репозиторий с сравнением обученных моделей и алгоритмов
- [ ] https://github.com/brightmart/text_classification

**Comment by nikibor** [comment]
Градиентный спуск:
1. Лекция/конспект:
- [ ] http://www.cs.pomona.edu/~dkauchak/classes/f16/cs158-f16/lectures/lecture9-gradient_descent.pdf
2.Stochastic Gradient Descent Methods for large scale pattern
classification
- [ ] http://stochastic.csa.iisc.ernet.in/~saswata/documents/termreport.pdf
3. Немного про применение механизма в оптимизации:
- [ ] http://projekter.aau.dk/projekter/files/208552317/main.pdf
4. Классификация испанского:
- [ ] https://www.uni-weimar.de/medien/webis/events/pan-13/pan13-papers-final/pan13-author-profiling/caurceldiaz13-notebook.pdf

**Comment by nikibor** [comment]
Сверточные сети:
1. Обзор и применение в классификации текста:
- [ ] https://pdfs.semanticscholar.org/presentation/d550/8f330772b00d751e49c31d66ba6f04b51267.pdf
2. Medical Text Classification using Convolutional Neural Networks
- [ ] https://arxiv.org/pdf/1704.06841.pdf
3. Combining Knowledge with Deep Convolutional Neural Networks for Short Text Classification
- [ ] https://www.ijcai.org/proceedings/2017/0406.pdf
4. Recurrent Convolutional Neural Networks for Text Classification 
- [ ] Скачана, на диске

**Comment by nikibor** [comment]
Рекурентные сети:
1. Recurrent Neural Network for Text Classification with Multi-Task Learning
- [ ] https://www.ijcai.org/Proceedings/16/Papers/408.pdf
2. Text Classification Improved by Integrating Bidirectional LSTM with Two-dimensional Max Pooling
- [ ] http://www.aclweb.org/anthology/C16-1329

**Comment by nikibor** [comment]
Слои нейронных сетей:
- [ ] http://www.cs.toronto.edu/%7Ersalakhu/papers/srivastava14a.pdf
